scripts/models/ngram_model/ngram_lm.py

```diff
@@ -0,0 +1,250 @@
+"""This module implements a ngram language model."""
```

Comment:
This is just a detail but we could provide more info on what is included here (perhaps that you can use Ngram or UniGram, what you can do with them etc)